tests/trainer/test_lr_finder.py

```diff
@@ -271,3 +271,22 @@ def test_lr_finder_fails_fast_on_bad_config(tmpdir):
     trainer = Trainer(default_root_dir=tmpdir, max_steps=2, auto_lr_find=True)
     with pytest.raises(MisconfigurationException, match='should have one of these fields'):
         trainer.tune(BoringModel())
+
+
+def test_lr_find_with_bs_scale(tmpdir):
+    """ Test that lr_find runs with batch_size_scaling """
+
+    hparams = EvalModelTemplate.get_default_hparams()
+    before_lr = hparams.get('learning_rate')
+    model = EvalModelTemplate(**hparams)
```

Comment:
Can we use `BoringModel` instead? It's much faster

Otherwise `ClassificationModel`